### ISSUE TITLE
undefined method `type' for nil:NilClass

### ISSUE THREAD

**Issue by Chwistophe** [solidusio-contrib/solidus_globalize#17]
Hello everyone, 

I'm getting an error trying when trying to setup a fresh install of solidus 1.4 and the `solidus_globalize` gem when trying to add translations to a Taxon from the backend (`http://localhost:3002/en/admin/taxonomies/1/translations`)

Environment : Ruby 2.1.7 / Rails 4.2.6 / Solidus 1.4
(also getting the same error using Ruby 2.3.1 / Rails 5.0.0.1 / Solidus 2.0)

I'm getting an `undefined method 'type' for nil:NilClass` error. Reading the manual I thought that it would could come from existing products that were removed but I'm also getting the same error with an empty database.

Here's my error : 

```
Showing /Users/christophe/.rvm/gems/ruby-2.1.7/bundler/gems/solidus_globalize-a662d4d1c551/app/views/spree/admin/translations/_form_fields.html.erb where line #23 raised:

undefined method `type' for nil:NilClass
Extracted source (around line #23):
21
22 <div class="panel-body">
23  <% if @resource.class.columns_hash[attr.to_s].type == :text %>
24    <%= g.text_area attr, class: 'form-control', rows: 4 %>
25  <% else %>
26 <%= g.text_field attr, class: 'form-control' %>

Trace of template inclusion: /Users/christophe/.rvm/gems/ruby-2.1.7/bundler/gems/solidus_globalize-a662d4d1c551/app/views/spree/admin/translations/_form.html.erb, /Users/christophe/.rvm/gems/ruby-2.1.7/bundler/gems/solidus_globalize-a662d4d1c551/app/views/spree/admin/translations/taxonomy.html.erb

Rails.root: /Users/christophe/Development/rails/perso/solitest

Application Trace | Framework Trace | Full Trace
/Users/christophe/.rvm/gems/ruby-2.1.7/bundler/gems/solidus_globalize-a662d4d1c551/app/views/spree/admin/translations/_form_fields.html.erb:23:in `block (3 levels) in _af3ea948e581e27342abd36c5913104a'
/Users/christophe/.rvm/gems/ruby-2.1.7/bundler/gems/solidus_globalize-a662d4d1c551/app/views/spree/admin/translations/_form_fields.html.erb:4:in `each'
/Users/christophe/.rvm/gems/ruby-2.1.7/bundler/gems/solidus_globalize-a662d4d1c551/app/views/spree/admin/translations/_form_fields.html.erb:4:in `each_with_index'
/Users/christophe/.rvm/gems/ruby-2.1.7/bundler/gems/solidus_globalize-a662d4d1c551/app/views/spree/admin/translations/_form_fields.html.erb:4:in `block (2 levels) in _af3ea948e581e27342abd36c5913104a'
actionview (4.2.6) lib/action_view/helpers/capture_helper.rb:38:in `block in capture'
actionview (4.2.6) lib/action_view/helpers/capture_helper.rb:202:in `with_output_buffer'
actionview (4.2.6) lib/action_view/helpers/capture_helper.rb:38:in `capture'
actionview (4.2.6) lib/action_view/helpers/form_helper.rb:714:in `fields_for'
/Users/christophe/.rvm/gems/ruby-2.1.7/bundler/gems/solidus_globalize-a662d4d1c551/config/initializers/form_builder_globalize_patch.rb:49:in `globalize_fields_for'
/Users/christophe/.rvm/gems/ruby-2.1.7/bundler/gems/solidus_globalize-a662d4d1c551/app/views/spree/admin/translations/_form_fields.html.erb:3:in `block in _af3ea948e581e27342abd36c5913104a'
/Users/christophe/.rvm/gems/ruby-2.1.7/bundler/gems/solidus_globalize-a662d4d1c551/app/views/spree/admin/translations/_form_fields.html.erb:2:in `each'
/Users/christophe/.rvm/gems/ruby-2.1.7/bundler/gems/solidus_globalize-a662d4d1c551/app/views/spree/admin/translations/_form_fields.html.erb:2:in `_af3ea948e581e27342abd36c5913104a'
actionview (4.2.6) lib/action_view/template.rb:145:in `block in render'
activesupport (4.2.6) lib/active_support/notifications.rb:166:in `instrument'
actionview (4.2.6) lib/action_view/template.rb:333:in `instrument'
actionview (4.2.6) lib/action_view/template.rb:143:in `render'
deface (1.0.2) lib/deface/action_view_extensions.rb:41:in `render'
actionview (4.2.6) lib/action_view/renderer/partial_renderer.rb:339:in `render_partial'
actionview (4.2.6) lib/action_view/renderer/partial_renderer.rb:310:in `block in render'
actionview (4.2.6) lib/action_view/renderer/abstract_renderer.rb:39:in `block in instrument'
activesupport (4.2.6) lib/active_support/notifications.rb:164:in `block in instrument'
activesupport (4.2.6) lib/active_support/notifications/instrumenter.rb:20:in `instrument'
activesupport (4.2.6) lib/active_support/notifications.rb:164:in `instrument'
actionview (4.2.6) lib/action_view/renderer/abstract_renderer.rb:39:in `instrument'
actionview (4.2.6) lib/action_view/renderer/partial_renderer.rb:309:in `render'
actionview (4.2.6) lib/action_view/renderer/renderer.rb:51:in `render_partial'
actionview (4.2.6) lib/action_view/helpers/rendering_helper.rb:35:in `render'
/Users/christophe/.rvm/gems/ruby-2.1.7/bundler/gems/solidus_globalize-a662d4d1c551/app/views/spree/admin/translations/_form.html.erb:9:in `block in _5eecd06f0115b28bcc029e89459e1d56'
actionview (4.2.6) lib/action_view/helpers/capture_helper.rb:38:in `block in capture'
actionview (4.2.6) lib/action_view/helpers/capture_helper.rb:202:in `with_output_buffer'
actionview (4.2.6) lib/action_view/helpers/capture_helper.rb:38:in `capture'
actionview (4.2.6) lib/action_view/helpers/form_helper.rb:444:in `form_for'
/Users/christophe/.rvm/gems/ruby-2.1.7/bundler/gems/solidus_globalize-a662d4d1c551/app/views/spree/admin/translations/_form.html.erb:8:in `_5eecd06f0115b28bcc029e89459e1d56'
actionview (4.2.6) lib/action_view/template.rb:145:in `block in render'
activesupport (4.2.6) lib/active_support/notifications.rb:166:in `instrument'
actionview (4.2.6) lib/action_view/template.rb:333:in `instrument'
actionview (4.2.6) lib/action_view/template.rb:143:in `render'
deface (1.0.2) lib/deface/action_view_extensions.rb:41:in `render'
actionview (4.2.6) lib/action_view/renderer/partial_renderer.rb:339:in `render_partial'
actionview (4.2.6) lib/action_view/renderer/partial_renderer.rb:310:in `block in render'
actionview (4.2.6) lib/action_view/renderer/abstract_renderer.rb:39:in `block in instrument'
activesupport (4.2.6) lib/active_support/notifications.rb:164:in `block in instrument'
activesupport (4.2.6) lib/active_support/notifications/instrumenter.rb:20:in `instrument'
activesupport (4.2.6) lib/active_support/notifications.rb:164:in `instrument'
actionview (4.2.6) lib/action_view/renderer/abstract_renderer.rb:39:in `instrument'
actionview (4.2.6) lib/action_view/renderer/partial_renderer.rb:309:in `render'
actionview (4.2.6) lib/action_view/renderer/renderer.rb:51:in `render_partial'
actionview (4.2.6) lib/action_view/helpers/rendering_helper.rb:35:in `render'
/Users/christophe/.rvm/gems/ruby-2.1.7/bundler/gems/solidus_globalize-a662d4d1c551/app/views/spree/admin/translations/taxonomy.html.erb:9:in `_1a7c4c4a8ae27da07f6dc207e5280fe8'
actionview (4.2.6) lib/action_view/template.rb:145:in `block in render'
activesupport (4.2.6) lib/active_support/notifications.rb:166:in `instrument'
actionview (4.2.6) lib/action_view/template.rb:333:in `instrument'
actionview (4.2.6) lib/action_view/template.rb:143:in `render'
deface (1.0.2) lib/deface/action_view_extensions.rb:41:in `render'
actionview (4.2.6) lib/action_view/renderer/template_renderer.rb:54:in `block (2 levels) in render_template'
actionview (4.2.6) lib/action_view/renderer/abstract_renderer.rb:39:in `block in instrument'
activesupport (4.2.6) lib/active_support/notifications.rb:164:in `block in instrument'
activesupport (4.2.6) lib/active_support/notifications/instrumenter.rb:20:in `instrument'
activesupport (4.2.6) lib/active_support/notifications.rb:164:in `instrument'
actionview (4.2.6) lib/action_view/renderer/abstract_renderer.rb:39:in `instrument'
actionview (4.2.6) lib/action_view/renderer/template_renderer.rb:53:in `block in render_template'
actionview (4.2.6) lib/action_view/renderer/template_renderer.rb:61:in `render_with_layout'
actionview (4.2.6) lib/action_view/renderer/template_renderer.rb:52:in `render_template'
actionview (4.2.6) lib/action_view/renderer/template_renderer.rb:14:in `render'
actionview (4.2.6) lib/action_view/renderer/renderer.rb:46:in `render_template'
actionview (4.2.6) lib/action_view/renderer/renderer.rb:27:in `render'
actionview (4.2.6) lib/action_view/rendering.rb:100:in `_render_template'
actionpack (4.2.6) lib/action_controller/metal/streaming.rb:217:in `_render_template'
actionview (4.2.6) lib/action_view/rendering.rb:83:in `render_to_body'
actionpack (4.2.6) lib/action_controller/metal/rendering.rb:32:in `render_to_body'
actionpack (4.2.6) lib/action_controller/metal/renderers.rb:37:in `render_to_body'
actionpack (4.2.6) lib/abstract_controller/rendering.rb:25:in `render'
actionpack (4.2.6) lib/action_controller/metal/rendering.rb:16:in `render'
actionpack (4.2.6) lib/action_controller/metal/instrumentation.rb:44:in `block (2 levels) in render'
activesupport (4.2.6) lib/active_support/core_ext/benchmark.rb:12:in `block in ms'
/Users/christophe/.rvm/rubies/ruby-2.1.7/lib/ruby/2.1.0/benchmark.rb:294:in `realtime'
activesupport (4.2.6) lib/active_support/core_ext/benchmark.rb:12:in `ms'
actionpack (4.2.6) lib/action_controller/metal/instrumentation.rb:44:in `block in render'
actionpack (4.2.6) lib/action_controller/metal/instrumentation.rb:87:in `cleanup_view_runtime'
activerecord (4.2.6) lib/active_record/railties/controller_runtime.rb:25:in `cleanup_view_runtime'
actionpack (4.2.6) lib/action_controller/metal/instrumentation.rb:43:in `render'
/Users/christophe/.rvm/gems/ruby-2.1.7/bundler/gems/solidus_globalize-a662d4d1c551/app/controllers/spree/admin/translations_controller.rb:11:in `index'
actionpack (4.2.6) lib/action_controller/metal/implicit_render.rb:4:in `send_action'
actionpack (4.2.6) lib/abstract_controller/base.rb:198:in `process_action'
actionpack (4.2.6) lib/action_controller/metal/rendering.rb:10:in `process_action'
actionpack (4.2.6) lib/abstract_controller/callbacks.rb:20:in `block in process_action'
activesupport (4.2.6) lib/active_support/callbacks.rb:117:in `call'
activesupport (4.2.6) lib/active_support/callbacks.rb:117:in `call'
activesupport (4.2.6) lib/active_support/callbacks.rb:555:in `block (2 levels) in compile'
activesupport (4.2.6) lib/active_support/callbacks.rb:505:in `call'
activesupport (4.2.6) lib/active_support/callbacks.rb:505:in `call'
activesupport (4.2.6) lib/active_support/callbacks.rb:92:in `__run_callbacks__'
activesupport (4.2.6) lib/active_support/callbacks.rb:778:in `_run_process_action_callbacks'
activesupport (4.2.6) lib/active_support/callbacks.rb:81:in `run_callbacks'
actionpack (4.2.6) lib/abstract_controller/callbacks.rb:19:in `process_action'
actionpack (4.2.6) lib/action_controller/metal/rescue.rb:29:in `process_action'
actionpack (4.2.6) lib/action_controller/metal/instrumentation.rb:32:in `block in process_action'
activesupport (4.2.6) lib/active_support/notifications.rb:164:in `block in instrument'
activesupport (4.2.6) lib/active_support/notifications/instrumenter.rb:20:in `instrument'
activesupport (4.2.6) lib/active_support/notifications.rb:164:in `instrument'
actionpack (4.2.6) lib/action_controller/metal/instrumentation.rb:30:in `process_action'
actionpack (4.2.6) lib/action_controller/metal/params_wrapper.rb:250:in `process_action'
activerecord (4.2.6) lib/active_record/railties/controller_runtime.rb:18:in `process_action'
actionpack (4.2.6) lib/abstract_controller/base.rb:137:in `process'
actionview (4.2.6) lib/action_view/rendering.rb:30:in `process'
actionpack (4.2.6) lib/action_controller/metal.rb:196:in `dispatch'
actionpack (4.2.6) lib/action_controller/metal/rack_delegation.rb:13:in `dispatch'
actionpack (4.2.6) lib/action_controller/metal.rb:237:in `block in action'
actionpack (4.2.6) lib/action_dispatch/routing/route_set.rb:74:in `call'
actionpack (4.2.6) lib/action_dispatch/routing/route_set.rb:74:in `dispatch'
actionpack (4.2.6) lib/action_dispatch/routing/route_set.rb:43:in `serve'
actionpack (4.2.6) lib/action_dispatch/journey/router.rb:43:in `block in serve'
actionpack (4.2.6) lib/action_dispatch/journey/router.rb:30:in `each'
actionpack (4.2.6) lib/action_dispatch/journey/router.rb:30:in `serve'
actionpack (4.2.6) lib/action_dispatch/routing/route_set.rb:817:in `call'
railties (4.2.6) lib/rails/engine.rb:518:in `call'
railties (4.2.6) lib/rails/railtie.rb:194:in `public_send'
railties (4.2.6) lib/rails/railtie.rb:194:in `method_missing'
actionpack (4.2.6) lib/action_dispatch/routing/mapper.rb:51:in `serve'
actionpack (4.2.6) lib/action_dispatch/journey/router.rb:43:in `block in serve'
actionpack (4.2.6) lib/action_dispatch/journey/router.rb:30:in `each'
actionpack (4.2.6) lib/action_dispatch/journey/router.rb:30:in `serve'
actionpack (4.2.6) lib/action_dispatch/routing/route_set.rb:817:in `call'
versioncake (3.2.0) lib/versioncake/rack/middleware.rb:17:in `call'
warden (1.2.6) lib/warden/manager.rb:35:in `block in call'
warden (1.2.6) lib/warden/manager.rb:34:in `catch'
warden (1.2.6) lib/warden/manager.rb:34:in `call'
rack (1.6.4) lib/rack/etag.rb:24:in `call'
rack (1.6.4) lib/rack/conditionalget.rb:25:in `call'
rack (1.6.4) lib/rack/head.rb:13:in `call'
actionpack (4.2.6) lib/action_dispatch/middleware/params_parser.rb:27:in `call'
actionpack (4.2.6) lib/action_dispatch/middleware/flash.rb:260:in `call'
rack (1.6.4) lib/rack/session/abstract/id.rb:225:in `context'
rack (1.6.4) lib/rack/session/abstract/id.rb:220:in `call'
actionpack (4.2.6) lib/action_dispatch/middleware/cookies.rb:560:in `call'
activerecord (4.2.6) lib/active_record/query_cache.rb:36:in `call'
activerecord (4.2.6) lib/active_record/connection_adapters/abstract/connection_pool.rb:653:in `call'
activerecord (4.2.6) lib/active_record/migration.rb:377:in `call'
actionpack (4.2.6) lib/action_dispatch/middleware/callbacks.rb:29:in `block in call'
activesupport (4.2.6) lib/active_support/callbacks.rb:88:in `__run_callbacks__'
activesupport (4.2.6) lib/active_support/callbacks.rb:778:in `_run_call_callbacks'
activesupport (4.2.6) lib/active_support/callbacks.rb:81:in `run_callbacks'
actionpack (4.2.6) lib/action_dispatch/middleware/callbacks.rb:27:in `call'
actionpack (4.2.6) lib/action_dispatch/middleware/reloader.rb:73:in `call'
actionpack (4.2.6) lib/action_dispatch/middleware/remote_ip.rb:78:in `call'
actionpack (4.2.6) lib/action_dispatch/middleware/debug_exceptions.rb:17:in `call'
web-console (2.3.0) lib/web_console/middleware.rb:28:in `block in call'
web-console (2.3.0) lib/web_console/middleware.rb:18:in `catch'
web-console (2.3.0) lib/web_console/middleware.rb:18:in `call'
actionpack (4.2.6) lib/action_dispatch/middleware/show_exceptions.rb:30:in `call'
railties (4.2.6) lib/rails/rack/logger.rb:38:in `call_app'
railties (4.2.6) lib/rails/rack/logger.rb:20:in `block in call'
activesupport (4.2.6) lib/active_support/tagged_logging.rb:68:in `block in tagged'
activesupport (4.2.6) lib/active_support/tagged_logging.rb:26:in `tagged'
activesupport (4.2.6) lib/active_support/tagged_logging.rb:68:in `tagged'
railties (4.2.6) lib/rails/rack/logger.rb:20:in `call'
request_store (1.3.1) lib/request_store/middleware.rb:9:in `call'
actionpack (4.2.6) lib/action_dispatch/middleware/request_id.rb:21:in `call'
rack (1.6.4) lib/rack/methodoverride.rb:22:in `call'
rack (1.6.4) lib/rack/runtime.rb:18:in `call'
activesupport (4.2.6) lib/active_support/cache/strategy/local_cache_middleware.rb:28:in `call'
rack (1.6.4) lib/rack/lock.rb:17:in `call'
actionpack (4.2.6) lib/action_dispatch/middleware/static.rb:120:in `call'
rack (1.6.4) lib/rack/sendfile.rb:113:in `call'
railties (4.2.6) lib/rails/engine.rb:518:in `call'
railties (4.2.6) lib/rails/application.rb:165:in `call'
rack (1.6.4) lib/rack/lock.rb:17:in `call'
rack (1.6.4) lib/rack/content_length.rb:15:in `call'
rack (1.6.4) lib/rack/handler/webrick.rb:88:in `service'
/Users/christophe/.rvm/rubies/ruby-2.1.7/lib/ruby/2.1.0/webrick/httpserver.rb:138:in `service'
/Users/christophe/.rvm/rubies/ruby-2.1.7/lib/ruby/2.1.0/webrick/httpserver.rb:94:in `run'
/Users/christophe/.rvm/rubies/ruby-2.1.7/lib/ruby/2.1.0/webrick/server.rb:295:in `block in start_thread'
Request

Parameters:

{"resource"=>"taxonomies",
 "resource_id"=>"1",
 "locale"=>"en"}
Toggle session dump
Toggle env dump
Response

Headers:

None
```

Debugging variables using the web console : 

```
>>  @resource.class
=> Spree::Taxonomy(id: integer, name: string, created_at: datetime, updated_at: datetime, position: integer)

>>  @resource.class.columns_hash
=> {"id"=>#<ActiveRecord::ConnectionAdapters::PostgreSQLColumn:0x007faa9ff5a220 @array=false, @name="id", @cast_type=#<ActiveRecord::ConnectionAdapters::PostgreSQL::OID::Integer:0x007faa9cabfa60 @precision=nil, @scale=nil, @limit=nil, @range=-2147483648...2147483648>, @sql_type="integer", @null=false, @default=nil, @default_function="nextval('spree_taxonomies_id_seq'::regclass)">, "created_at"=>#<ActiveRecord::ConnectionAdapters::PostgreSQLColumn:0x007faa9ff59f78 @array=false, @name="created_at", @cast_type=#<ActiveRecord::ConnectionAdapters::PostgreSQL::OID::DateTime:0x007faa9cabe930 @precision=nil, @scale=nil, @limit=nil>, @sql_type="timestamp without time zone", @null=true, @default=nil, @default_function=nil>, "updated_at"=>#<ActiveRecord::ConnectionAdapters::PostgreSQLColumn:0x007faa9ff59de8 @array=false, @name="updated_at", @cast_type=#<ActiveRecord::ConnectionAdapters::PostgreSQL::OID::DateTime:0x007faa9cabe930 @precision=nil, @scale=nil, @limit=nil>, @sql_type="timestamp without time zone", @null=true, @default=nil, @default_function=nil>, "position"=>#<ActiveRecord::ConnectionAdapters::PostgreSQLColumn:0x007faa9ff59c80 @array=false, @name="position", @cast_type=#<ActiveRecord::ConnectionAdapters::PostgreSQL::OID::Integer:0x007faa9cabfa60 @precision=nil, @scale=nil, @limit=nil, @range=-2147483648...2147483648>, @sql_type="integer", @null=true, @default="0", @default_function=nil>}

>>  attr
=> :name

```

Here's my Gemfile :

```
source "https://rubygems.org"
gem "rails", "4.2.6"
gem "pg", "~> 0.15"
gem "sass-rails", "~> 5.0"
gem "uglifier", ">= 1.3.0"
gem "coffee-rails", "~> 4.1.0"

gem "solidus"
gem "solidus_auth_devise"
gem "solidus_i18n", github: "solidusio-contrib/solidus_i18n"
gem "globalize", github: "globalize/globalize"
gem "solidus_globalize", github: "solidusio-contrib/solidus_globalize", branch: "master"

gem "jquery-rails"
gem "turbolinks"
gem "jbuilder", "~> 2.0"
gem "sdoc", "~> 0.4.0", group: :doc
```

Here are my steps to reproduce : 
(I use spree:install options to avoid load_samples so I don't get empty translations errors from Globalize)

```
rails g spree:install --migrate=false --sample=false --seed=false
bundle exec rake railties:install:migrations
bundle exec rake db:migrate
bundle exec rake db:seed
bundle exec rails g solidus:auth:install
bin/rails g solidus_globalize:install
```

Then added : 

```
Spree.config do |config|
  SolidusI18n::Config.available_locales = [:en, :fr] # displayed on frontend select box
  SolidusGlobalize::Config.supported_locales = [:en, :fr] # displayed on translation forms
  ...
```

to `lib/initializers/spree.rb`

Thanks for your help! 💯

**Comment by ghost** [comment]
Hi,
We were getting the same error in all translations views in backend.
To resolve temporarily this issue, we added a deface, replacing the line 23 of app/views/spree/admin/translations/_form_fields.html.erb

`<% if @resource.class.columns_hash[attr.to_s].type == :text %>`

for  

`<% if @resource.translations.table.engine.columns_hash[attr.to_s].type == :text %>`

because translatables columns don't exists in class.columns_hash.
But we don't know if this is the best solution.

**Comment by Chwistophe** [comment]
@sergi2Be thanks!
Can't wait for this gem to be updated :)

**Comment by recyclablestudios** [comment]
@sergi2Be 
How can I do that?
I can't find app/views/spree/admin/translations/_form_fields.html.erb

**Comment by AndreiMotinga** [comment]
@recyclablestudios [here's that partial ](https://github.com/solidusio-contrib/solidus_globalize/blob/master/app/views/spree/admin/translations/_form_fields.html.erb)
copy it to you project. 

@sergi2Be 's solution didn't work for my so I just replaced
`<% if @resource.class.columns_hash[attr.to_s].type == :text %>`
with
`<% if @resource.class.columns_hash[attr.to_s].try(:type) == :text %>`

**Comment by AndreiMotinga** [comment]
@ChristopheCholot,  should be fixed with #34

**Comment by hdl-tangier** [comment]
Using `column_for_attribute` fixed this error for me but got some errors with javascript when adding locales to the selector. Any updates?

```
translations.self-7bcdcc6….js?body=1:12 Uncaught TypeError: Cannot read property 'replace' of undefined
    at fn (translations.self-7bcdcc6….js?body=1:12)
    at display_locale_fields (translations.self-7bcdcc6….js?body=1:26)
    at HTMLSelectElement.<anonymous> (translations.self-7bcdcc6….js?body=1:44)
    at HTMLSelectElement.dispatch (jquery.self-bd7ddd3….js?body=1:5227)
    at HTMLSelectElement.elemData.handle (jquery.self-bd7ddd3….js?body=1:4879)
    at Object.trigger (jquery.self-bd7ddd3….js?body=1:5131)
    at HTMLSelectElement.<anonymous> (jquery.self-bd7ddd3….js?body=1:5861)
    at Function.each (jquery.self-bd7ddd3….js?body=1:371)
    at jQuery.fn.init.each (jquery.self-bd7ddd3….js?body=1:138)
    at jQuery.fn.init.trigger (jquery.self-bd7ddd3….js?body=1:5860)
```

**Comment by alepore** [comment]
This should be fixed by #54
